### ISSUE TITLE
Open articles directly in the browser

### ISSUE THREAD

**Issue by Saectar** [hello-efficiency-inc/raven-reader#644]
I can't find a way to open the article I'm reading directly in my browser. It is possible to copy the link manually, but it would be more useful to do it from a button.

I find it convenient to be able to open certain articles in the browser so I can comment on them or translate them into my native language (although it would also be helpful if Raven could translate articles directly from the program).

Please add this functionality.

Thanks

**Comment by MyNameIsTroll** [comment]
I agree, I often look for the button to open the article in my browser but I have to copy it then open the browser and paste the link. It's tedious and not very practical.

**Comment by lykahb** [comment]
On Ubuntu 22.04 the "Copy link" button does not work

**Comment by holzi-x** [comment]
This used to be implemented - see [commit 66829d0](https://github.com/hello-efficiency-inc/raven-reader/commit/66829d099415220440e6c309c92d403f87512200) and this [conversation](https://github.com/hello-efficiency-inc/raven-reader/pull/4). Don't know where this got lost, but will search it later
